### ISSUE TITLE
[bug] `accelerate` not found with runpod "ashleykza/kohya:1.11.1"

### ISSUE THREAD

**Issue by xih** [ashleykleynhans/kohya-docker#6]
While running a runpod container with this setup, I'm getting an `accelerate not found`. Any tips to debug? 

![CleanShot 2024-01-18 at 18 52 11@2x](https://github.com/ashleykleynhans/kohya-docker/assets/8076886/2bbc64d0-ae3c-4888-a5e4-da0bcc41cc35)

ERROR:
![CleanShot 2024-01-18 at 18 53 02@2x](https://github.com/ashleykleynhans/kohya-docker/assets/8076886/595b42a9-fed7-4cef-a820-5bef9a896c20)

Full logs below
```
04:39:07-748602 INFO     Start training LoRA Standard ...                       
04:39:07-750284 INFO     Checking for duplicate image filenames in training data
                         directory...                                           
04:39:07-752158 INFO     Valid image folder names found in:                     
                         /workspace/organize/watches/img                        
04:39:07-753621 INFO     Headless mode, skipping verification if model already  
                         exist... if model already exist it will be             
                         overwritten...                                         
04:39:07-755396 INFO     Folder 20_wxwatch watch: 24 images found               
04:39:07-756743 INFO     Folder 20_wxwatch watch: 480 steps                     
04:39:07-758027 INFO     Total steps: 480                                       
04:39:07-759290 INFO     Train batch size: 2                                    
04:39:07-759959 INFO     Gradient accumulation steps: 1                         
04:39:07-760572 INFO     Epoch: 10                                              
04:39:07-761151 INFO     Regulatization factor: 1                               
04:39:07-761755 INFO     max_train_steps (480 / 2 / 1 * 10 * 1) = 2400          
04:39:07-762504 INFO     stop_text_encoder_training = 0                         
04:39:07-763121 INFO     lr_warmup_steps = 240                                  
04:39:07-763757 INFO     Can't use LR warmup with LR Scheduler constant...      
                         ignoring...                                            
04:39:07-764482 INFO     Saving training config to                              
                         /workspace/organize/watches/model/wxwatches_20240119-04
                         3907.json...                                           
04:39:07-765433 INFO     accelerate launch --num_cpu_threads_per_process=2      
                         "./sdxl_train_network.py" --enable_bucket              
                         --min_bucket_reso=256 --max_bucket_reso=2048           
                         --pretrained_model_name_or_path="stabilityai/stable-dif
                         fusion-xl-base-1.0"                                    
                         --train_data_dir="/workspace/organize/watches/img"     
                         --resolution="1024,1024"                               
                         --output_dir="/workspace/organize/watches/model"       
                         --logging_dir="/workspace/organize/watches/log"        
                         --network_alpha="1" --save_model_as=safetensors        
                         --network_module=networks.lora --text_encoder_lr=5e-05 
                         --unet_lr=0.0001 --network_dim=8                       
                         --output_name="wxwatches"                              
                         --lr_scheduler_num_cycles="10" --no_half_vae           
                         --learning_rate="5e-05" --lr_scheduler="constant"      
                         --train_batch_size="2" --max_train_steps="2400"        
                         --save_every_n_epochs="1" --mixed_precision="fp16"     
                         --save_precision="fp16" --cache_latents                
                         --cache_latents_to_disk --optimizer_type="Adafactor"   
                         --optimizer_args scale_parameter=False                 
                         relative_step=False warmup_init=False                  
                         --max_grad_norm="1" --max_data_loader_n_workers="0"    
                         --bucket_reso_steps=64 --xformers --bucket_no_upscale  
                         --noise_offset=0.0                                     
/bin/sh: 1: accelerate: not found
```

**Comment by leetxen** [comment]
Trying to troubleshoot this also.  trying to get it to setup accelerate, or something off google searches, nothing is working so far.

**Comment by leetxen** [comment]
I was able to get past this issue @xih , I did the following
```
cd workspace/kohya_ss/
./setup.sh
```
After that I was able to start the training.  at least it got past to the next error for me.  good luck.

**Comment by ashleykleynhans** [comment]
My humblest apologies. This was an oversight on my part, I tried to fix a bug in the `fix_venv.sh` script so that it works with Network volumes on RunPod but introduced a new bug in the process whereby it doesn't correctly fix a venv that does not reside at `/workspace/venv`.  I am fixing it and will push a fix shortly.

**Comment by ashleykleynhans** [comment]
This has now been resolved in the 1.11.2 release.

**Comment by leetxen** [comment]
@ashleykleynhans no problem.  I want to say thank you for this docker image.  I have been trying for 2 weeks to get a lora trained on a windows machine.  Your project got me over the problems and I am training my first one, should be done in a few hours.  After that I will redownload the container and give it another try and report back.
